Foosball/Foosball/Tournament_team_selection.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Foosball
+{
+    public partial class Tournament_team_selection : Form
+    {
+        public Tournament_team_selection(int selectedMode)
+        {
+            InitializeComponent();
+            textBoxRemover(selectedMode);
+        }
+
+        private void back_Click(object sender, EventArgs e)
+        {
+            this.Hide();
+            var load = new Tournament_mode();
+            load.ShowDialog();
+            this.Close();
+        }
+
+        private void textBoxRemover(int amount)
+        {
+            TextBox[] textBox = new TextBox[16];
+            for (int i = amount; i < 16; i++) {
```

Comment:
use

for 
{

}

or 

for {

}

everywhere, to maintain code readability